### PR TITLE
chore: align asset/dataset terminology

### LINKED ISSUE #187: change `asset` to `dataset`
We once introduced the term "asset". As we aligned with the DCAT terminology, the wording should be consistent.

### PR DESCRIPTION
**Pull request by juliapampus**
Closes #187 

Changed `asset` to `dataset`. Removed occurences of `IDS`.

### CODE REVIEW

**Review by jimmarino — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -25,12 +25,12 @@ __*dataspace*__.
 
 ## Introduction
 
-Sharing data between autonomous entities requires the provision of metadata to facilitate the transfer of assets by making use of a data transfer (or application layer) protocol.
+Sharing data between autonomous entities requires the provision of metadata to facilitate the transfer of datasets by making use of a data transfer (or application layer) protocol.
 The __Dataspace Protocol__ defines how this metadata is provisioned:
 
-1. How data assets are deployed as [DCAT Catalogs](https://www.w3.org/TR/vocab-dcat-3/) and usage control is expressed as [ODRL Policies](https://www.w3.org/TR/odrl-model/).
+1. How data datasets are deployed as [DCAT Catalogs](https://www.w3.org/TR/vocab-dcat-3/) and usage control is expressed as [ODRL Policies](https://www.w3.org/TR/odrl-model/).
```
Comment (mkollenstart) [inline README.md:31]: Should be dataset instead of data datasets

File: catalog/catalog.protocol.md [hunk 2]
```diff
@@ -112,24 +112,24 @@ The catalog service may require an authorization token. Details for including th
...
 
 #### 3.1.1 odrl:hasPolicy
 
-An asset entry Dataset must have 1..N `hasPolicy` attributes that contain an ODRL `Offer` defining the usage control policy associated with the asset. Offers must NOT contain any
-target attributes. The target of an offer is the asset associated with the containing asset entry.
+A `Dataset` must have 1..N `hasPolicy` attributes that contain an ODRL `Offer` defining the usage control policy associated with the dataset. Offers must NOT contain any
+target attributes. The target of an offer is the associated dataset.
 
-> Note: As `odrl:hasPolicy rdfs:domain odrl:Asset` and `AssetEntry isA dcat:Dataset`, each `Asset Entry` is also an `odrl:Asset` from an ODRL perspective.
+> Note: As `odrl:hasPolicy rdfs:domain odrl:Asset` and `dspace:Dataset isA dcat:Dataset`, each `Dataset` is also an `odrl:Asset` from an ODRL perspective.
```
Comment (mkollenstart) [inline catalog/catalog.protocol.md:124]: We don't define the class `dspace:Dataset` but only the property `dspace:dataset`, so I suggest removing the "and `dspace:Dataset isA dcat:Dataset`" part

File: model/terminology.md [hunk 3]
```diff
@@ -35,38 +35,38 @@ If a  trusted technology system is required that records and verifies those doma
...
 
-## Asset
+## Dataset
 
 Data or a technical service that can be shared by a `Participant`.
 
 ## Policy
 
-A set of rules, duties, and obligations that define the terms of use for an `Asset`.
+A set of rules, duties, and obligations that define the terms of use for an `Dataset`.
```
Comment (mkollenstart) [inline model/terminology.md:44]: a instead of an

File: model/terminology.md [hunk 4]
```diff
@@ -35,38 +35,38 @@ If a  trusted technology system is required that records and verifies those doma
...
 
 ## Contract Negotiation
 
 A set of interactions between a provider `Connector` and consumer `Connector` that establish an `Agreement`.
 
-## Asset Transfer
+## Dataset Transfer
 
-A set of interactions between a provider `Connector` and consumer `Connector` that give access to an `Asset` under the terms of an `Agreement`.
+A set of interactions between a provider `Connector` and consumer `Connector` that give access to an `Dataset` under the terms of an `Agreement`.
```
Comment (mkollenstart) [inline model/terminology.md:72]: a instead of an

**Review by mkollenstart — COMMENTED** [review]
Some minor comments

File: model/model.md [hunk 5]
```diff
@@ -16,24 +16,24 @@ Note that all relationships are multiplicities unless specified.
...
-  [DCAT DataService](https://www.w3.org/TR/vocab-dcat-3/#Class:Data_Service) that references a `Connector` where assets may be obtained.
-- An `Asset Entry` has **one** `Offer`, which is an [ODRL Offer](https://www.w3.org/TR/odrl-model/#policy-offer) describing the _usage control policy_ associated with the asset.
-- A `Connector` is a `Participant Agent` that performs `Contract Negotiation` and `Asset Transfer` operations with another connector. An outcome of a `ContractNegotiation` may
-  be the production of an `Agreement`, which is an [ODRL Agreement](https://www.w3.org/TR/odrl-model/#policy-agreement) defining the _usage control policy_ agreed to for an asset.
+- A `Catalog` contains one or more `Datasets`, which are [DCAT Datasets](https://www.w3.org/TR/vocab-dcat-3/#Class:Dataset). A `Catalog` also contains **_at least one_**
+  [DCAT DataService](https://www.w3.org/TR/vocab-dcat-3/#Class:Data_Service) that references a `Connector` where datasets may be obtained.
+- An `Dataset` has **one** `Offer`, which is an [ODRL Offer](https://www.w3.org/TR/odrl-model/#policy-offer) describing the _usage control policy_ associated with the dataset.
```
Comment (matgnt) [inline model/model.md:34]: Is this correct? To my understanding a `dataset` would contain multiple `offers` which is what I can find in `hasPolicy`. That was at least to conclusion when I asked this last time:
https://github.com/International-Data-Spaces-Association/ids-specification/issues/76#issuecomment-1493778662
We also updated the examples accordingly here:
https://github.com/International-Data-Spaces-Association/ids-specification/pull/80/files

File: model/model.md [hunk 6]
```diff
@@ -48,18 +48,18 @@ The classes and definitions used in the dataspace protocol are reused from diffe
...
 
-### 2.2.2 Asset Entry
+### 2.2.2 Dataset
 
-An `Asset Entry` is a [DCAT Dataset](https://www.w3.org/TR/vocab-dcat-3/#Class:Dataset) with the following attributes:
+A `Dataset` is a [DCAT Dataset](https://www.w3.org/TR/vocab-dcat-3/#Class:Dataset) with the following attributes:
 
-- 1..N `hasPolicy` attributes that contain an ODRL `Offer` defining the usage control policy associated with the asset. **_Offers must NOT contain any target attributes. The
-  target of an offer is the asset associated with the containing asset entry._** (ODRL PROFILE)
+- 1..N `hasPolicy` attributes that contain an ODRL `Offer` defining the usage control policy associated with the dataset. **_Offers must NOT contain any target attributes. The
```
Comment (matgnt) [inline model/model.md:59]: .... continuing from above... also this indicates multiple offers to my understanding.

**Review by matgnt — COMMENTED** [review]
Added questions regarding number of offers per dataset.

**Review by mkollenstart — APPROVED** [review]

File: model/model.md (hunk 5 above)
Comment (matgnt) [reply model/model.md]: I just did a quick test with 2 EDCs. When I create 2 different policies and 2 contract definitions (with the same asset...) it produces the expected catalog result with a list of `hasPolicy` containing exactly the 2 previously created policies.

To my understanding the sentence needs to be updated to:

```
An `Dataset` has **multiple** `Offers`, which are a [ODRL Offer](https://www.w3.org/TR/odrl-model/#policy-offer) describing the _usage control policy_ associated with the dataset.
```

**Comment by matgnt** [comment]
That was fast @juliapampus ;-)  you already updated the relevant part :-) Thanks!

**Review by matgnt — APPROVED** [review]

**Pull request merged**